boefjes/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN --mount=type=cache,target=/root/.cache \
     pip install --upgrade pip \
     && pip install -r /tmp/boefjes-requirements.txt \
     && rm /tmp/boefjes-requirements.txt \
+    && pip install -r requirements.txt \
```

Comment:
Could also be an `else` the line below although I don't expect much performance difference

Reply:
We could, but it's done this way in all of our Dockerfiles and looks readable this way.